doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1096 @@
+  * Forwarding of :c:enum:`SENSOR_EVT_MOVEMENT_ACTIVITY_DETECTED` and :c:enum:`SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED` events by the sensor module.
+  * :ref:`Real-time configurations <real_time_configs>`, which can now configure the upper and lower thresholds for motion detection.
+    You can also configure the timeout after which the sensor reports inactivity.
+    It is now set to 30 seconds.
+  * LwM2M schema.
+    To use the new parameters, upload the updated :file:`config_object_descript.xml` file to AVSystem.
+  * The conversions of RSRP and RSRQ.
+    These now use common macros that follow the conversion algorithms defined in the `AT Commands Reference Guide`_.
+  * Bootstrapping to be disabled by default.
+    This allows connecting to the default LwM2M service AVSystem's `Coiote Device Management`_ using free tier accounts.
+  * ``CONFIG_DATA_DEVICE_MODE`` Kconfig option to be a choice that can be set to either ``CONFIG_DATA_DEVICE_MODE_ACTIVE`` or ``CONFIG_DATA_DEVICE_MODE_PASSIVE``, depending on the desired device mode.
+  * The default sample timeout for sample requests that include neighbor cell searches.
+    The time is now set to 11 seconds.
+  * Documentation structure.
+
+* Fixed:
+
+  * An issue that reports GNSS altitude, accuracy, and speed incorrectly when using LwM2M engine.
+  * An issue that caused modem FOTA jobs to be reported as not validated to nRF Cloud.
+
+* Removed:
+
+  * ``CONFIG_APP_REQUEST_GNSS_ON_INITIAL_SAMPLING`` Kconfig option.
+  * ``CONFIG_APP_REQUEST_NEIGHBOR_CELLS_DATA`` Kconfig option.
+  * ``CONFIG_EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO`` Kconfig option.
+  * ``CONFIG_MODEM_CONVERT_RSRP_AND_RSPQ_TO_DB`` Kconfig option.
+  * ``CONFIG_DATA_ACCELEROMETER_THRESHOLD`` Kconfig option.
+  * ``CONFIG_DATA_ACCELEROMETER_BUFFER_STORE`` Kconfig option.
+  * ``CONFIG_DATA_ACCELEROMETER_BUFFER_COUNT`` Kconfig option.
+  * ``SENSOR_EVT_MOVEMENT_DATA_READY`` event.
+
+nRF9160: Serial LTE modem
+-------------------------
+
+* Added:
+
+  * URC for GNSS timeout sleep event.
+  * Selected flags support in ``#XRECV`` and ``#XRECVFROM`` commands.
+  * Multi-PDN support in the Socket service.
+  * New ``#XGPSDEL`` command to delete GNSS data from non-volatile memory.
+  * New ``#XDFUSIZE`` command to get the size of the DFU file image.
+
+* Updated:
+
+  * The application to use :ref:`TF-M <ug_tfm>` enabled by default.
+  * The GNSS service to signify location information to nRF Cloud.
+  * The AT response and the URC sent when the application enters and exits data mode.
+  * ``WAKEUP_PIN`` and ``INTERFACE_PIN`` to be now defined as *Active Low*.
+    Both are *High* when the SLM application starts.
+
+* Fixed an issue where the features of the mbedTLS v3.1 were not enabled by default, which caused the native TLS to not work.
+  The documentation of the :ref:`SLM_AT_SOCKET`' socket option numbers was updated accordingly.
```

Comment:
@pirun , @FrancescoSer , please review and approve entry for #8707

Reply:
Btw, the correct spelling is Mbed TLS.